site/src/components/SignInForm/SignInForm.tsx

```diff
@@ -126,10 +127,7 @@ export const SignInForm: FC<React.PropsWithChildren<SignInFormProps>> = ({
       </Maybe>
 
       <Maybe condition={!passwordEnabled && !oAuthEnabled}>
-        <AlertBanner
-          severity="error"
-          text="No authentication methods configured!"
-        />
+        <Alert severity="error">No authentication methods configured!</Alert>
```

Comment:
Related to my comment above, what is the difference between these two ways of displaying errors?

Reply:
I answered in the previous comment.